diff -urN wwwoffle-2.6d.old/Makefile wwwoffle-2.6d/Makefile
--- wwwoffle-2.6d.old/Makefile	Tue Jul 17 11:22:01 2001
+++ wwwoffle-2.6d/Makefile	Tue Jul 17 11:43:08 2001
@@ -20,7 +20,9 @@
 # For UNIX.
 INSTDIR=/usr/local
 SPOOLDIR=/var/spool/wwwoffle
+SPOOLDIR_BUILD=/var/spool/wwwoffle
 CONFDIR=/var/spool/wwwoffle
+CONFDIR_BUILD=/var/spool/wwwoffle
 MANDIR=$(INSTDIR)/man
 DOCDIR=$(INSTDIR)/doc/wwwoffle
 
@@ -356,55 +358,55 @@
 	  done )
 
 install_cache :
-	@[ ! -d $(SPOOLDIR) ] || [ ! -d $(SPOOLDIR)/outgoing ] || [ -d $(SPOOLDIR)/prevtime1 ] || \
+	@[ ! -d $(SPOOLDIR_BUILD) ] || [ ! -d $(SPOOLDIR_BUILD)/outgoing ] || [ -d $(SPOOLDIR_BUILD)/prevtime1 ] || \
 	 (echo "WWWOFFLE: " ;\
 	  echo "WWWOFFLE: Your existing cache looks earlier than version 2.2, cannot continue." ;\
 	  echo "WWWOFFLE: Read the file NEWS for details." ;\
 	  echo "WWWOFFLE: " ;\
 	  exit 1 )
-	[ -x $(SPOOLDIR) ] || $(INSTALL) -d -m 750 $(SPOOLDIR)
-	[ -x $(SPOOLDIR)/http ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/http
-	[ -x $(SPOOLDIR)/outgoing ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/outgoing
-	[ -x $(SPOOLDIR)/monitor ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/monitor
-	[ -x $(SPOOLDIR)/lasttime ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/lasttime
-	[ -x $(SPOOLDIR)/prevtime1 ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/prevtime1
+	[ -x $(SPOOLDIR_BUILD) ] || $(INSTALL) -d -m 750 $(SPOOLDIR_BUILD)
+	[ -x $(SPOOLDIR_BUILD)/http ] || $(INSTALL) -d -m 750 $(SPOOLDIR_BUILD)/http
+	[ -x $(SPOOLDIR_BUILD)/outgoing ] || $(INSTALL) -d -m 750 $(SPOOLDIR_BUILD)/outgoing
+	[ -x $(SPOOLDIR_BUILD)/monitor ] || $(INSTALL) -d -m 750 $(SPOOLDIR_BUILD)/monitor
+	[ -x $(SPOOLDIR_BUILD)/lasttime ] || $(INSTALL) -d -m 750 $(SPOOLDIR_BUILD)/lasttime
+	[ -x $(SPOOLDIR_BUILD)/prevtime1 ] || $(INSTALL) -d -m 750 $(SPOOLDIR_BUILD)/prevtime1
 	for file in outgoing/* ; do \
-	  $(INSTALL) -c -m 644 $$file $(SPOOLDIR)/outgoing ;\
+	  $(INSTALL) -c -m 644 $$file $(SPOOLDIR_BUILD)/outgoing ;\
 	done
 
 install_html : html
-	@[ ! -x $(SPOOLDIR)/html.old ] || \
+	@[ ! -x $(SPOOLDIR_BUILD)/html.old ] || \
 	 (echo "WWWOFFLE: " ;\
-	  echo "WWWOFFLE: There is already a directory $(SPOOLDIR)/html.old." ;\
+	  echo "WWWOFFLE: There is already a directory $(SPOOLDIR_BUILD)/html.old." ;\
 	  echo "WWWOFFLE: Remove it and re-run make." ;\
 	  echo "WWWOFFLE: " ;\
 	  exit 1 )
-	[ ! -x $(SPOOLDIR)/html ] || mv $(SPOOLDIR)/html $(SPOOLDIR)/html.old
-	[ -x $(SPOOLDIR)/html ] || mkdir $(SPOOLDIR)/html
-	tar cf $(SPOOLDIR)/html.tar html && cd $(SPOOLDIR) && tar xpf html.tar && rm html.tar
+	[ ! -x $(SPOOLDIR_BUILD)/html ] || mv $(SPOOLDIR_BUILD)/html $(SPOOLDIR_BUILD)/html.old
+	[ -x $(SPOOLDIR_BUILD)/html ] || mkdir $(SPOOLDIR_BUILD)/html
+	tar cf $(SPOOLDIR_BUILD)/html.tar html && cd $(SPOOLDIR_BUILD) && tar xpf html.tar && rm html.tar
 	[ "x$(LANG)" = "x" ] || [ ! -d lang-$(LANG) ] || ( cd lang-$(LANG) && \
-	  tar cf $(SPOOLDIR)/html.tar html && cd $(SPOOLDIR) && tar xpf html.tar && rm html.tar )
-	cd $(SPOOLDIR)/html && ./fixup-install.sh $(SPOOLDIR) $(LOCALHOST) && rm ./fixup-install.sh
+	  tar cf $(SPOOLDIR_BUILD)/html.tar html && cd $(SPOOLDIR_BUILD) && tar xpf html.tar && rm html.tar )
+	cd $(SPOOLDIR_BUILD)/html && ./fixup-install.sh $(SPOOLDIR_BUILD) $(LOCALHOST) && rm ./fixup-install.sh
 # Now fix the permissions that tar preserved, we needed to use 'tar xpf' to get round root's umask.
 # These two will fail unless you are root, that is OK because the owner is already you.
-	-chown -R 0 $(SPOOLDIR)/html > /dev/null 2>&1
-	-chgrp -R 0 $(SPOOLDIR)/html > /dev/null 2>&1
+	-chown -R 0 $(SPOOLDIR_BUILD)/html > /dev/null 2>&1
+	-chgrp -R 0 $(SPOOLDIR_BUILD)/html > /dev/null 2>&1
 
 install_config :
 	sed -e 's%SPOOLDIR%$(SPOOLDIR)%' -e 's%CONFDIR%$(CONFDIR)%' < wwwoffle.conf > wwwoffle.conf.install
-	[ -x $(CONFDIR) ] || $(INSTALL) -d -m 750 $(CONFDIR)
-	@[ ! -f $(CONFDIR)/wwwoffle.conf ] || \
+	[ -x $(CONFDIR_BUILD) ] || $(INSTALL) -d -m 750 $(CONFDIR_BUILD)
+	@[ ! -f $(CONFDIR_BUILD)/wwwoffle.conf ] || \
 	 (echo "WWWOFFLE: " ;\
-	  echo "WWWOFFLE: There is already a config file $(CONFDIR)/wwwoffle.conf." ;\
-	  echo "WWWOFFLE: Run 'upgrade-config.pl $(CONFDIR)/wwwoffle.conf' to upgrade it." ;\
+	  echo "WWWOFFLE: There is already a config file $(CONFDIR_BUILD)/wwwoffle.conf." ;\
+	  echo "WWWOFFLE: Run 'upgrade-config.pl $(CONFDIR_BUILD)/wwwoffle.conf' to upgrade it." ;\
 	  echo "WWWOFFLE: " )
-	[  ! -f $(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR)/wwwoffle.conf.install
-	[    -f $(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR)/wwwoffle.conf
+	[  ! -f $(CONFDIR_BUILD)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR_BUILD)/wwwoffle.conf.install
+	[    -f $(CONFDIR_BUILD)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR_BUILD)/wwwoffle.conf
 
 install_fixup-win32:
 	for file in $(DOCDIR)/* ; do \
 	  [ -f $$file ] && awk '{print $$0 "\r";}' < $$file > $$file.txt && rm -f $$file ;\
 	done
-	rm -f $(SPOOLDIR)/html/index.html && ln $(SPOOLDIR)/html/Welcome.html $(SPOOLDIR)/html/index.html
+	rm -f $(SPOOLDIR_BUILD)/html/index.html && ln $(SPOOLDIR_BUILD)/html/Welcome.html $(SPOOLDIR_BUILD)/html/index.html
 
 ########
