diff -urN wwwoffle-2.6b.old/Makefile wwwoffle-2.6b/Makefile
--- wwwoffle-2.6b.old/Makefile	Sat Mar 24 15:02:31 2001
+++ wwwoffle-2.6b/Makefile	Sat Apr 21 14:04:01 2001
@@ -20,7 +20,9 @@
 # For UNIX.
 INSTDIR=/usr/local
 SPOOLDIR=/var/spool/wwwoffle
+INST_SPOOLDIR=/var/spool/wwwoffle
 CONFDIR=/var/spool/wwwoffle
+INST_CONFDIR=/var/spool/wwwoffle
 MANDIR=$(INSTDIR)/man
 DOCDIR=$(INSTDIR)/doc/wwwoffle
 
@@ -341,50 +343,50 @@
 	  done )
 
 install_cache :
-	@[ ! -d $(SPOOLDIR) -o -d $(SPOOLDIR)/prevtime1 ] || \
+	@[ ! -d $(INST_SPOOLDIR) -o -d $(INST_SPOOLDIR)/prevtime1 ] || \
 	 (echo "WWWOFFLE: " ;\
 	  echo "WWWOFFLE: Your existing cache looks earlier than version 2.2, cannot continue." ;\
 	  echo "WWWOFFLE: Read the file NEWS for details." ;\
 	  echo "WWWOFFLE: " ;\
 	  exit 1 )
-	[ -x $(SPOOLDIR) ] || $(INSTALL) -d -m 750 $(SPOOLDIR)
-	[ -x $(SPOOLDIR)/http ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/http
-	[ -x $(SPOOLDIR)/outgoing ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/outgoing
-	[ -x $(SPOOLDIR)/monitor ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/monitor
-	[ -x $(SPOOLDIR)/lasttime ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/lasttime
-	[ -x $(SPOOLDIR)/prevtime1 ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/prevtime1
+	[ -x $(INST_SPOOLDIR) ] || $(INSTALL) -d -m 750 $(INST_SPOOLDIR)
+	[ -x $(INST_SPOOLDIR)/http ] || $(INSTALL) -d -m 750 $(INST_SPOOLDIR)/http
+	[ -x $(INST_SPOOLDIR)/outgoing ] || $(INSTALL) -d -m 750 $(INST_SPOOLDIR)/outgoing
+	[ -x $(INST_SPOOLDIR)/monitor ] || $(INSTALL) -d -m 750 $(INST_SPOOLDIR)/monitor
+	[ -x $(INST_SPOOLDIR)/lasttime ] || $(INSTALL) -d -m 750 $(INST_SPOOLDIR)/lasttime
+	[ -x $(INST_SPOOLDIR)/prevtime1 ] || $(INSTALL) -d -m 750 $(INST_SPOOLDIR)/prevtime1
 	for file in outgoing/* ; do \
-	  $(INSTALL) -c -m 644 $$file $(SPOOLDIR)/outgoing ;\
+	  $(INSTALL) -c -m 644 $$file $(INST_SPOOLDIR)/outgoing ;\
 	done
 
 install_html : html
-	@[ ! -x $(SPOOLDIR)/html.old ] || \
+	@[ ! -x $(INST_SPOOLDIR)/html.old ] || \
 	 (echo "WWWOFFLE: " ;\
-	  echo "WWWOFFLE: There is already a directory $(SPOOLDIR)/html.old." ;\
+	  echo "WWWOFFLE: There is already a directory $(INST_SPOOLDIR)/html.old." ;\
 	  echo "WWWOFFLE: Remove it and re-run make." ;\
 	  echo "WWWOFFLE: " ;\
 	  exit 1 )
-	[ ! -x $(SPOOLDIR)/html ] || mv $(SPOOLDIR)/html $(SPOOLDIR)/html.old
-	[ -x $(SPOOLDIR)/html ] || mkdir $(SPOOLDIR)/html
-	tar cf $(SPOOLDIR)/html.tar html && cd $(SPOOLDIR) && tar xpf html.tar && rm html.tar
+	[ ! -x $(INST_SPOOLDIR)/html ] || mv $(INST_SPOOLDIR)/html $(INST_SPOOLDIR)/html.old
+	[ -x $(INST_SPOOLDIR)/html ] || mkdir $(INST_SPOOLDIR)/html
+	tar cf $(INST_SPOOLDIR)/html.tar html && cd $(INST_SPOOLDIR) && tar xpf html.tar && rm html.tar
 	[ "x$(LANG)" = "x" ] || [ ! -d lang-$(LANG) ] || ( cd lang-$(LANG) && \
-	  tar cf $(SPOOLDIR)/html.tar html && cd $(SPOOLDIR) && tar xpf html.tar && rm html.tar )
-	cd $(SPOOLDIR)/html && ./fixup-install.sh $(SPOOLDIR) $(LOCALHOST) && rm ./fixup-install.sh
+	  tar cf $(INST_SPOOLDIR)/html.tar html && cd $(INST_SPOOLDIR) && tar xpf html.tar && rm html.tar )
+	cd $(INST_SPOOLDIR)/html && ./fixup-install.sh $(INST_SPOOLDIR) $(LOCALHOST) && rm ./fixup-install.sh
 # Now fix the permissions that tar preserved, we needed to use 'tar xpf' to get round root's umask.
 # These two will fail unless you are root, that is OK because the owner is already you.
-	-chown -R 0 $(SPOOLDIR)/html > /dev/null 2>&1
-	-chgrp -R 0 $(SPOOLDIR)/html > /dev/null 2>&1
+	-chown -R 0 $(INST_SPOOLDIR)/html > /dev/null 2>&1
+	-chgrp -R 0 $(INST_SPOOLDIR)/html > /dev/null 2>&1
 
 install_config :
 	sed -e 's%SPOOLDIR%$(SPOOLDIR)%' -e 's%CONFDIR%$(CONFDIR)%' < wwwoffle.conf > wwwoffle.conf.install
-	[ -x $(CONFDIR) ] || $(INSTALL) -d -m 750 $(CONFDIR)
-	@[ ! -f $(CONFDIR)/wwwoffle.conf ] || \
+	[ -x $(INST_CONFDIR) ] || $(INSTALL) -d -m 750 $(INST_CONFDIR)
+	@[ ! -f $(INST_CONFDIR)/wwwoffle.conf ] || \
 	 (echo "WWWOFFLE: " ;\
-	  echo "WWWOFFLE: There is already a config file $(CONFDIR)/wwwoffle.conf." ;\
-	  echo "WWWOFFLE: Run 'upgrade-config.pl $(CONFDIR)/wwwoffle.conf' to upgrade it." ;\
+	  echo "WWWOFFLE: There is already a config file $(INST_CONFDIR)/wwwoffle.conf." ;\
+	  echo "WWWOFFLE: Run 'upgrade-config.pl $(INST_CONFDIR)/wwwoffle.conf' to upgrade it." ;\
 	  echo "WWWOFFLE: " )
-	[  ! -f $(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR)/wwwoffle.conf.install
-	[    -f $(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR)/wwwoffle.conf
+	[  ! -f $(INST_CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(INST_CONFDIR)/wwwoffle.conf.install
+	[    -f $(INST_CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(INST_CONFDIR)/wwwoffle.conf
 
 install_fixup-win32:
 	for file in $(DOCDIR)/* ; do \
