diff -urN wwwoffle-2.6b.old/html/local/dontget/replacement.html wwwoffle-2.6b/html/local/dontget/replacement.html
--- wwwoffle-2.6b.old/html/local/dontget/replacement.html	Thu Jan  1 01:00:00 1970
+++ wwwoffle-2.6b/html/local/dontget/replacement.html	Sat Apr 21 10:20:13 2001
@@ -0,0 +1,2 @@
+<HTML>
+</HTML>
\ No newline at end of file
diff -urN wwwoffle-2.6b.old/wwwoffle.conf wwwoffle-2.6b/wwwoffle.conf
--- wwwoffle-2.6b.old/wwwoffle.conf	Mon Mar 19 20:11:43 2001
+++ wwwoffle-2.6b/wwwoffle.conf	Sat Apr 21 10:19:15 2001
@@ -654,7 +654,7 @@
 # <*://*/*.gif> replacement = /local/dontget/replacement.gif
 # <*://*/*.png> replacement = /local/dontget/replacement.png
 # <*://*/*.js>  replacement = /local/dontget/replacement.js
-# replacement = /local/dontget/replacement.gif
+# replacement = /local/dontget/replacement.html
 
 #### Example ####
 # Don't get from any hosts in the barfoo.com domain.
