diff -urN wwwoffle-2.7a.old/cache/Makefile.in wwwoffle-2.7a/cache/Makefile.in
--- wwwoffle-2.7a.old/cache/Makefile.in	Sat Mar  2 17:20:01 2002
+++ wwwoffle-2.7a/cache/Makefile.in	Tue Apr  9 10:58:01 2002
@@ -65,56 +65,56 @@
 ######## PROPER INSTALLATION ########
 
 install-mkdir :
-	@[ ! -d $(SPOOLDIR) ] || [ ! -d $(SPOOLDIR)/outgoing ] || [ -d $(SPOOLDIR)/prevtime1 ] || \
+	@[ ! -d $(DESTDIR)$(SPOOLDIR) ] || [ ! -d $(DESTDIR)$(SPOOLDIR)/outgoing ] || [ -d $(DESTDIR)$(SPOOLDIR)/prevtime1 ] || \
 	 (echo "WWWOFFLE: " ;\
 	  echo "WWWOFFLE: Your existing cache looks earlier than version 2.2, cannot continue." ;\
 	  echo "WWWOFFLE: Read the file NEWS for details." ;\
 	  echo "WWWOFFLE: " ;\
 	  exit 1 )
-	[ -x $(SPOOLDIR) ]           || $(INSTALL) -d -m 750 $(SPOOLDIR)
-	[ -x $(SPOOLDIR)/http ]      || $(INSTALL) -d -m 750 $(SPOOLDIR)/http
-	[ -x $(SPOOLDIR)/outgoing ]  || $(INSTALL) -d -m 750 $(SPOOLDIR)/outgoing
-	[ -x $(SPOOLDIR)/monitor ]   || $(INSTALL) -d -m 750 $(SPOOLDIR)/monitor
-	[ -x $(SPOOLDIR)/lasttime ]  || $(INSTALL) -d -m 750 $(SPOOLDIR)/lasttime
-	[ -x $(SPOOLDIR)/prevtime1 ] || $(INSTALL) -d -m 750 $(SPOOLDIR)/prevtime1
-	[ -x $(SPOOLDIR)/lastout ]   || $(INSTALL) -d -m 750 $(SPOOLDIR)/lastout
-	[ -x $(SPOOLDIR)/local ]     || $(INSTALL) -d -m 750 $(SPOOLDIR)/local
+	[ -x $(DESTDIR)$(SPOOLDIR) ]           || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)
+	[ -x $(DESTDIR)$(SPOOLDIR)/http ]      || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)/http
+	[ -x $(DESTDIR)$(SPOOLDIR)/outgoing ]  || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)/outgoing
+	[ -x $(DESTDIR)$(SPOOLDIR)/monitor ]   || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)/monitor
+	[ -x $(DESTDIR)$(SPOOLDIR)/lasttime ]  || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)/lasttime
+	[ -x $(DESTDIR)$(SPOOLDIR)/prevtime1 ] || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)/prevtime1
+	[ -x $(DESTDIR)$(SPOOLDIR)/lastout ]   || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)/lastout
+	[ -x $(DESTDIR)$(SPOOLDIR)/local ]     || $(INSTALL) -d -m 750 $(DESTDIR)$(SPOOLDIR)/local
 
 install-outgoing : install-mkdir
 	for file in outgoing/* ; do \
-	  $(INSTALL) -c -m 644 $$file $(SPOOLDIR)/outgoing ;\
+	  $(INSTALL) -c -m 644 $$file $(DESTDIR)$(SPOOLDIR)/outgoing ;\
 	done
 
 install-html : install-mkdir
-	@[ ! -x $(SPOOLDIR)/html.old ] || \
+	@[ ! -x $(DESTDIR)$(SPOOLDIR)/html.old ] || \
 	 (echo "WWWOFFLE: " ;\
-	  echo "WWWOFFLE: There is already a directory $(SPOOLDIR)/html.old." ;\
+	  echo "WWWOFFLE: There is already a directory $(DESTDIR)$(SPOOLDIR)/html.old." ;\
 	  echo "WWWOFFLE: Remove it and re-run make." ;\
 	  echo "WWWOFFLE: " ;\
 	  exit 1 )
-	[ ! -x $(SPOOLDIR)/html ] || mv $(SPOOLDIR)/html $(SPOOLDIR)/html.old
+	[ ! -x $(DESTDIR)$(SPOOLDIR)/html ] || mv $(DESTDIR)$(SPOOLDIR)/html $(DESTDIR)$(SPOOLDIR)/html.old
 	cd html ; rm -f default ; ln -sf $(DEFLANG) default
-	$(TAR) cf $(SPOOLDIR)/html.tar html && cd $(SPOOLDIR) && $(TAR) xpf html.tar && rm html.tar
-	cd $(SPOOLDIR)/html && ./fixup-install.sh $(LOCALHOST):$(HTTP_PORT) && rm fixup-install.sh
+	$(TAR) cf $(DESTDIR)$(SPOOLDIR)/html.tar html && cd $(DESTDIR)$(SPOOLDIR) && $(TAR) xpf html.tar && rm html.tar
+	cd $(DESTDIR)$(SPOOLDIR)/html && ./fixup-install.sh $(LOCALHOST):$(HTTP_PORT) && rm fixup-install.sh
 # Now fix the ownerships that tar preserved, we needed to use 'tar xpf' to get round root's umask.
 # These two will fail unless you are root, that is OK because the owner is already you.
-	-chown -R 0 $(SPOOLDIR)/html > /dev/null 2>&1
-	-chgrp -R 0 $(SPOOLDIR)/html > /dev/null 2>&1
+	-chown -R 0 $(DESTDIR)$(SPOOLDIR)/html > /dev/null 2>&1
+	-chgrp -R 0 $(DESTDIR)$(SPOOLDIR)/html > /dev/null 2>&1
 
 install-html-win32 : install-html
 	rm -f $(SPOOLDIR)/html/index.html && ln $(SPOOLDIR)/html/Welcome.html $(SPOOLDIR)/html/index.html
 
 install-search : install-mkdir
-	@[ ! -x $(SPOOLDIR)/search.old ] || \
+	@[ ! -x $(DESTDIR)$(SPOOLDIR)/search.old ] || \
 	 (echo "WWWOFFLE: " ;\
-	  echo "WWWOFFLE: There is already a directory $(SPOOLDIR)/search.old." ;\
+	  echo "WWWOFFLE: There is already a directory $(DESTDIR)$(SPOOLDIR)/search.old." ;\
 	  echo "WWWOFFLE: Remove it and re-run make." ;\
 	  echo "WWWOFFLE: " ;\
 	  exit 1 )
-	[ ! -x $(SPOOLDIR)/search ] || mv $(SPOOLDIR)/search $(SPOOLDIR)/search.old
-	$(TAR) cf $(SPOOLDIR)/search.tar search && cd $(SPOOLDIR) && $(TAR) xpf search.tar && rm search.tar
-	cd $(SPOOLDIR)/search && ./fixup-install.sh $(SPOOLDIR) $(LOCALHOST):$(HTTP_PORT) && rm fixup-install.sh
+	[ ! -x $(DESTDIR)$(SPOOLDIR)/search ] || mv $(DESTDIR)$(SPOOLDIR)/search $(DESTDIR)$(SPOOLDIR)/search.old
+	$(TAR) cf $(DESTDIR)$(SPOOLDIR)/search.tar search && cd $(DESTDIR)$(SPOOLDIR) && $(TAR) xpf search.tar && rm search.tar
+	cd $(DESTDIR)$(SPOOLDIR)/search && ./fixup-install.sh $(DESTDIR)$(SPOOLDIR) $(LOCALHOST):$(HTTP_PORT) && rm fixup-install.sh
 # Now fix the permissions that tar preserved, we needed to use 'tar xpf' to get round root's umask.
 # These two will fail unless you are root, that is OK because the owner is already you.
-	-chown -R 0 $(SPOOLDIR)/search > /dev/null 2>&1
-	-chgrp -R 0 $(SPOOLDIR)/search > /dev/null 2>&1
+	-chown -R 0 $(DESTDIR)$(SPOOLDIR)/search > /dev/null 2>&1
+	-chgrp -R 0 $(DESTDIR)$(SPOOLDIR)/search > /dev/null 2>&1
diff -urN wwwoffle-2.7a.old/conf/Makefile.in wwwoffle-2.7a/conf/Makefile.in
--- wwwoffle-2.7a.old/conf/Makefile.in	Sun Sep 16 17:01:52 2001
+++ wwwoffle-2.7a/conf/Makefile.in	Tue Apr  9 10:57:04 2002
@@ -43,14 +43,14 @@
 
 install : compile
 	sed -e 's%SPOOLDIR%$(SPOOLDIR)%' -e 's%CONFDIR%$(CONFDIR)%' < wwwoffle.conf > wwwoffle.conf.install
-	[ -x $(CONFDIR) ] || $(INSTALL) -d -m 750 $(CONFDIR)
-	@[ ! -f $(CONFDIR)/wwwoffle.conf ] || \
+	[ -x $(DESTDIR)$(CONFDIR) ] || $(INSTALL) -d -m 750 $(DESTDIR)$(CONFDIR)
+	@[ ! -f $(DESTDIR)$(CONFDIR)/wwwoffle.conf ] || \
 	 (echo "WWWOFFLE: " ;\
-	  echo "WWWOFFLE: There is already a config file $(CONFDIR)/wwwoffle.conf." ;\
-	  echo "WWWOFFLE: Run 'conf/upgrade-config.pl $(CONFDIR)/wwwoffle.conf' to upgrade it." ;\
+	  echo "WWWOFFLE: There is already a config file $(DESTDIR)$(CONFDIR)/wwwoffle.conf." ;\
+	  echo "WWWOFFLE: Run 'conf/upgrade-config.pl $(DESTDIR)$(CONFDIR)/wwwoffle.conf' to upgrade it." ;\
 	  echo "WWWOFFLE: " )
-	[  ! -f $(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR)/wwwoffle.conf.install
-	[    -f $(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(CONFDIR)/wwwoffle.conf
+	[  ! -f $(DESTDIR)$(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(DESTDIR)$(CONFDIR)/wwwoffle.conf.install
+	[    -f $(DESTDIR)$(CONFDIR)/wwwoffle.conf ] || $(INSTALL) -c -m 640 wwwoffle.conf.install $(DESTDIR)$(CONFDIR)/wwwoffle.conf
 
 ######## CONF FILE ########
 
diff -urN wwwoffle-2.7a.old/doc/Makefile.in wwwoffle-2.7a/doc/Makefile.in
--- wwwoffle-2.7a.old/doc/Makefile.in	Wed Jan  2 17:14:32 2002
+++ wwwoffle-2.7a/doc/Makefile.in	Tue Apr  9 10:57:04 2002
@@ -51,28 +51,28 @@
 install-win32 : install-doc-win32
 
 install-man :
-	[ -x $(mandir) ] || $(INSTALL) -d -m 755 $(mandir)
-	[ -x $(mandir)/man1 ] || $(INSTALL) -d -m 755 $(mandir)/man1
-	$(INSTALL) -c -m 644 wwwoffle.man $(mandir)/man1/wwwoffle.1
-	[ -x $(mandir)/man5 ] || $(INSTALL) -d -m 755 $(mandir)/man5
+	[ -x $(DESTDIR)$(mandir) ] || $(INSTALL) -d -m 755 $(DESTDIR)$(mandir)
+	[ -x $(DESTDIR)$(mandir)/man1 ] || $(INSTALL) -d -m 755 $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -c -m 644 wwwoffle.man $(DESTDIR)$(mandir)/man1/wwwoffle.1
+	[ -x $(DESTDIR)$(mandir)/man5 ] || $(INSTALL) -d -m 755 $(DESTDIR)$(mandir)/man5
 	sed -e 's%SPOOLDIR%$(SPOOLDIR)%' -e 's%CONFDIR%$(CONFDIR)%' < wwwoffle.conf.man > wwwoffle.conf.man.install
-	$(INSTALL) -c -m 644 wwwoffle.conf.man.install $(mandir)/man5/wwwoffle.conf.5
-	[ -x $(mandir)/man8 ] || $(INSTALL) -d -m 755 $(mandir)/man8
-	$(INSTALL) -c -m 644 wwwoffled.man $(mandir)/man8/wwwoffled.8
+	$(INSTALL) -c -m 644 wwwoffle.conf.man.install $(DESTDIR)$(mandir)/man5/wwwoffle.conf.5
+	[ -x $(DESTDIR)$(mandir)/man8 ] || $(INSTALL) -d -m 755 $(DESTDIR)$(mandir)/man8
+	$(INSTALL) -c -m 644 wwwoffled.man $(DESTDIR)$(mandir)/man8/wwwoffled.8
 
 install-doc :
-	[ -x $(docdir) ] || $(INSTALL) -d -m 755 $(docdir)
+	[ -x $(DESTDIR)$(docdir) ] || $(INSTALL) -d -m 755 $(DESTDIR)$(docdir)
 	cd $(DEFLANG) ; \
 	  for file in CHANGES.CONF CONVERT COPYING FAQ INSTALL NEWS README README.* ; do \
-	    [ ! -f $$file ] || $(INSTALL) -c -m 644 $$file $(docdir)/$$file ;\
+	    [ ! -f $$file ] || $(INSTALL) -c -m 644 $$file $(DESTDIR)$(docdir)/$$file ;\
 	  done
 	for lang in $(LANGUAGES); do \
 	  [ ! -d $$lang -o $$lang = $(DEFLANG) ] || \
 	  ( cd $$lang && \
-	    $(INSTALL) -d -m 755 $(docdir)/$$lang && \
+	    $(INSTALL) -d -m 755 $(DESTDIR)$(docdir)/$$lang && \
 	    for file in * ; do \
 	      [ ! -f $$file -o $$file = wwwoffle.conf.man.template ] || \
-                $(INSTALL) -c -m 644 $$file $(docdir)/$$lang/$$file ;\
+                $(INSTALL) -c -m 644 $$file $(DESTDIR)$(docdir)/$$lang/$$file ;\
 	    done ) ;\
 	done
 
diff -urN wwwoffle-2.7a.old/src/Makefile.in wwwoffle-2.7a/src/Makefile.in
--- wwwoffle-2.7a.old/src/Makefile.in	Sat Mar  2 13:04:46 2002
+++ wwwoffle-2.7a/src/Makefile.in	Tue Apr  9 10:57:04 2002
@@ -75,17 +75,17 @@
 install : compile install-unix @install_win32@
 
 install-unix :
-	[ -x $(bindir) ] || $(INSTALL) -d -m 755 $(bindir)
-	$(INSTALL) -c -m 755 wwwoffle $(bindir)
-	$(INSTALL) -c -m 755 wwwoffle-tools $(bindir)
-	ln -sf wwwoffle-tools $(bindir)/wwwoffle-ls
-	ln -sf wwwoffle-tools $(bindir)/wwwoffle-mv
-	ln -sf wwwoffle-tools $(bindir)/wwwoffle-rm
-	ln -sf wwwoffle-tools $(bindir)/wwwoffle-read
-	ln -sf wwwoffle-tools $(bindir)/wwwoffle-write
-	ln -sf wwwoffle-tools $(bindir)/wwwoffle-hash
-	[ -x $(sbindir) ] || $(INSTALL) -d -m 755 $(sbindir)
-	$(INSTALL) -c -m 755 wwwoffled $(sbindir)
+	[ -x $(DESTDIR)$(bindir) ] || $(INSTALL) -d -m 755 $(DESTDIR)$(bindir)
+	$(INSTALL) -c -m 755 wwwoffle $(DESTDIR)$(bindir)
+	$(INSTALL) -c -m 755 wwwoffle-tools $(DESTDIR)$(bindir)
+	ln -sf wwwoffle-tools $(DESTDIR)$(bindir)/wwwoffle-ls
+	ln -sf wwwoffle-tools $(DESTDIR)$(bindir)/wwwoffle-mv
+	ln -sf wwwoffle-tools $(DESTDIR)$(bindir)/wwwoffle-rm
+	ln -sf wwwoffle-tools $(DESTDIR)$(bindir)/wwwoffle-read
+	ln -sf wwwoffle-tools $(DESTDIR)$(bindir)/wwwoffle-write
+	ln -sf wwwoffle-tools $(DESTDIR)$(bindir)/wwwoffle-hash
+	[ -x $(DESTDIR)$(sbindir) ] || $(INSTALL) -d -m 755 $(DESTDIR)$(sbindir)
+	$(INSTALL) -c -m 755 wwwoffled $(DESTDIR)$(sbindir)
 
 install-win32 :
 	[ -x $(bindir) ] || $(INSTALL) -d -m 755 $(bindir)
