diff -urN wwwoffle-2.6d.org/sockets6.c wwwoffle-2.6d/sockets6.c
--- wwwoffle-2.6d.org/sockets6.c	Sun Aug  5 00:23:05 2001
+++ wwwoffle-2.6d/sockets6.c	Sun Aug  5 00:23:25 2001
@@ -505,7 +505,7 @@
 static int getnameinfo_or_timeout(struct sockaddr *addr,int len,char **host,char **ip,char **port)
 {
  struct sigaction action;
- static char _host[NI_MAXHOST],_ip[46],_port[12];
+ static char _host[NI_MAXHOST],_ip[INET6_ADDRSTRLEN],_port[12];
 
  *host="(unknown)";
  *ip="(unknown)";
@@ -515,7 +515,7 @@
 
  if(!timeout_dns)
    {
-    gai_errno=getnameinfo(addr,len,_ip,NI_MAXHOST,_port,12,NI_NUMERICHOST|NI_NUMERICSERV);
+    gai_errno=getnameinfo(addr,len,_ip,INET6_ADDRSTRLEN,_port,12,NI_NUMERICHOST|NI_NUMERICSERV);
     if(gai_errno==0)
       {
        if(ip)
@@ -552,7 +552,7 @@
    }
  else
    {
-    gai_errno=getnameinfo(addr,len,_ip  ,NI_MAXHOST,_port,12,NI_NUMERICHOST|NI_NUMERICSERV);
+    gai_errno=getnameinfo(addr,len,_ip,INET6_ADDRSTRLEN,_port,12,NI_NUMERICHOST|NI_NUMERICSERV);
     if(gai_errno==0)
       {
        if(ip)
