diff -urN wwwoffle-2.5d.org/Makefile wwwoffle-2.5d/Makefile
--- wwwoffle-2.5d.org/Makefile	Wed Mar  1 18:04:39 2000
+++ wwwoffle-2.5d/Makefile	Wed Mar  1 18:28:11 2000
@@ -16,7 +16,8 @@
 
 LOCALHOST=localhost:8080
 
-INSTDIR=/usr/local
+DESTDIR=
+INSTDIR=$(DESTDIR)/usr/local
 SPOOLDIR=/var/spool/wwwoffle
 CONFDIR=/var/spool/wwwoffle
 
@@ -271,14 +272,14 @@
 	$(INSTALL) -c -m 755 cygwin1.dll $(INSTDIR)/bin
 
 install_doc :
-	[ -x $(INSTDIR)/man ] || $(INSTALL) -d -m 755 $(INSTDIR)/man
-	[ -x $(INSTDIR)/man/man1 ] || $(INSTALL) -d -m 755 $(INSTDIR)/man/man1
-	$(INSTALL) -c -m 644 wwwoffle.man $(INSTDIR)/man/man1/wwwoffle.1
-	[ -x $(INSTDIR)/man/man5 ] || $(INSTALL) -d -m 755 $(INSTDIR)/man/man5
-	sed -e 's%SPOOLDIR%$(SPOOLDIR)%' -e 's%CONFDIR%$(CONFDIR)%' < wwwoffle.conf.man > wwwoffle.conf.man.install
-	$(INSTALL) -c -m 644 wwwoffle.conf.man.install $(INSTDIR)/man/man5/wwwoffle.conf.5
-	[ -x $(INSTDIR)/man/man8 ] || $(INSTALL) -d -m 755 $(INSTDIR)/man/man8
-	$(INSTALL) -c -m 644 wwwoffled.man $(INSTDIR)/man/man8/wwwoffled.8
+	[ -x $(MANDIR) ] || $(INSTALL) -d -m 755 $(MANDIR)
+	[ -x $(MANDIR)/man1 ] || $(INSTALL) -d -m 755 $(MANDIR)/man1
+	$(INSTALL) -c -m 644 wwwoffle.man $(MANDIR)/man1/wwwoffle.1
+	[ -x $(MANDIR)/man5 ] || $(INSTALL) -d -m 755 $(MANDIR)/man5
+	sed -e 's#SPOOLDIR#$(SPOOLDIR)#' -e 's#CONFDIR#$(CONFDIR)#' < wwwoffle.conf.man > wwwoffle.conf.man.install
+	$(INSTALL) -c -m 644 wwwoffle.conf.man.install $(MANDIR)/man5/wwwoffle.conf.5
+	[ -x $(MANDIR)/man8 ] || $(INSTALL) -d -m 755 $(MANDIR)/man8
+	$(INSTALL) -c -m 644 wwwoffled.man $(MANDIR)/man8/wwwoffled.8
 	[ -x $(INSTDIR)/doc/wwwoffle ] || $(INSTALL) -d -m 755 $(INSTDIR)/doc/wwwoffle
 	for file in CHANGES.CONF CONVERT COPYING FAQ INSTALL NEWS README* UPGRADE ; do \
 	  $(INSTALL) -c -m 644 $$file $(INSTDIR)/doc/wwwoffle/$$file ;\
@@ -348,7 +349,7 @@
 	-chgrp -R 0 $(SPOOLDIR)/html > /dev/null 2>&1
 
 install_config :
-	sed -e 's%SPOOLDIR%$(SPOOLDIR)%' -e 's%CONFDIR%$(CONFDIR)%' < wwwoffle.conf > wwwoffle.conf.install
+	sed -e 's#SPOOLDIR#$(SPOOLDIR)#' -e 's#CONFDIR#$(CONFDIR)#' < wwwoffle.conf > wwwoffle.conf.install
 	[ -x $(CONFDIR) ] || $(INSTALL) -d -m 750 $(CONFDIR)
 	@[ ! -f $(CONFDIR)/wwwoffle.conf ] || \
 	 (echo "WWWOFFLE: " ;\
