--- wwwoffle-2.9/conf/wwwoffle.conf~	2006-04-02 16:52:55.000000000 +0200
+++ wwwoffle-2.9/conf/wwwoffle.conf	2006-04-06 00:16:03.000000000 +0200
@@ -123,7 +123,7 @@
 StartUp
 {
  bind-ipv4         = 0.0.0.0
- bind-ipv6         = ::
+ bind-ipv6         = none
 
  http-port         = 8080
  https-port        = 8443
@@ -131,8 +131,8 @@
 
  spool-dir         = SPOOLDIR
 
-#run-uid           = daemon
-#run-gid           = daemon
+run-uid           = wwwoffle
+run-gid           = wwwoffle
 
  use-syslog        = yes
 
--- wwwoffle-2.9/conf/wwwoffle.conf.template~	2006-04-06 00:13:12.000000000 +0200
+++ wwwoffle-2.9/conf/wwwoffle.conf.template	2006-04-06 00:16:54.000000000 +0200
@@ -12,7 +12,7 @@
 StartUp
 {
  bind-ipv4         = 0.0.0.0
- bind-ipv6         = ::
+ bind-ipv6         = none
 
  http-port         = 8080
  https-port        = 8443
@@ -20,8 +20,8 @@
 
  spool-dir         = SPOOLDIR
 
-#run-uid           = daemon
-#run-gid           = daemon
+run-uid           = wwwoffle
+run-gid           = wwwoffle
 
  use-syslog        = yes
 
