--- wwwoffle-2.9.a/src/certinfo.c-	2007-11-15 10:31:54.000000000 +0100
+++ wwwoffle-2.9a/src/certinfo.c	2007-11-15 10:32:41.000000000 +0100
@@ -543,7 +543,7 @@
 
 static void display_certificate(int fd,gnutls_x509_crt_t crt)
 {
- gnutls_datum_t xml={NULL,0};
+ gnutls_datum_t txt={NULL,0};
  char *dn,*issuer_dn;
  time_t activation,expiration;
  char *activation_str,*expiration_str;
@@ -642,9 +642,9 @@
  else
     key_ca="Error";
 
- /* XML formatted certificate. */
+ /* Formatted certificate. */
 
- gnutls_x509_crt_to_xml(crt,&xml,GNUTLS_XML_NORMAL);
+ gnutls_x509_crt_print (crt, GNUTLS_X509_CRT_FULL, &txt);
 
  /* Output the information. */
 
@@ -656,7 +656,7 @@
                  "key_algo",key_algo,
                  "key_usage",key_usage,
                  "key_ca",key_ca,
-                 "xml",xml.data,
+                 "info",txt.data,
                  NULL);
 
  /* Tidy up and exit */
@@ -664,7 +664,7 @@
  free(dn);
  free(issuer_dn);
 
- free(xml.data);
+ free(txt.data);
 }
